src/Request/Channel/OpenNowHtmlRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace StadGent\Services\OpeningHours\Request\Channel;
+
+use StadGent\Services\OpeningHours\Request\AcceptType;
+use StadGent\Services\OpeningHours\Request\MethodType;
+use StadGent\Services\OpeningHours\Request\RequestAbstract;
+
+/**
+ * Request to get OpenNow for a Channel in HTML format.
+ *
+ * @package StadGent\Services\OpeningHours\Request\Channel
+ */
+class OpenNowHtmlRequest extends RequestAbstract
+{
+    /**
+     * Get all channels for a service by the Service & Channel ID.
+     *
+     * @param int $serviceId
+     *   The Service ID to get the channel for.
+     * @param int $channelId
+     *   The Channel ID to get.
+     */
+    public function __construct($serviceId, $channelId)
+    {
+        $uri = sprintf(
+            'services/%d/channels/%d/open-now',
+            (int) $serviceId,
+            (int) $channelId
+        );
+
+        parent::__construct(
+            MethodType::GET,
+            $uri,
+            ['Accept' => AcceptType::HTML]
```

Comment:
As discussed this can be refactored to a property/constant

Reply:
Will be fixed in the refactoring PR.